README.md

```diff
@@ -0,0 +1,98 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Creative Commns Global Summit 2015
```

Comment:
[Optional] Your readme file looks great. I suggest that you use a heading that tells the title of the project.